### PR TITLE
Add capability to send collapsed groups (isExpanded=false flag, for dxDataGrid)

### PR DESCRIPTION
**Pull request by AlekseyMartynov**
Also, remote grouping enabled in Sample site

/cc @IlyaKhD

### CODE REVIEW

File: net/DevExtreme.AspNet.Data/GroupHelper.cs [hunk 1]
```diff
@@ -43,21 +48,31 @@ class GroupHelper<T> : ExpressionCompiler {
         }
 
 
-        IList<DevExtremeGroup> Group(IEnumerable<T> data, GroupingInfo groupInfo) {
+        IList<DevExtremeGroup> Group(IEnumerable<T> data, GroupingInfo groupInfo, bool isMostNested) {
```
Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data/GroupHelper.cs]: FYI this func will be reworked during integration with summary calculation

Comment (IlyaKhD) [inline net/DevExtreme.AspNet.Data/GroupHelper.cs]: Will be the `isMostedNested` param renamed? It's named a bit tricky

Comment (AlekseyMartynov) [inline net/DevExtreme.AspNet.Data/GroupHelper.cs]: Groups will be always populated, and then emptied after summary calc is done

**Comment by San4es** [comment]
:shipit:

**Pull request merged**